crates/edr_evm/src/test_utils.rs

```diff
@@ -394,6 +424,56 @@ pub async fn run_full_block<
     Ok(())
 }
 
+/// Forks from given block number to compare with locally generated header for
+/// that block
+pub async fn assert_replay_header<
```

Comment:
Question: Could you please explain the difference between this and the previous method?

Reply:
I edited the function documentation with the expected usage of the function, hope it's more clear now 🙂